### ISSUE TITLE
Take content out of tables on MGIB-SR rates page

### ISSUE THREAD

**Issue by RLHecht** [department-of-veterans-affairs/va.gov-team#42496]
## Issue Description
In continuing our effort to remove unnecessary tables from our pages for accessibility reasons, we need to take the content out of the tables on our draft Montgomery Gi Bill Selected Reserve rates page. 

Drupal preview: http://preview-prod.vfs.va.gov/preview?nodeId=5449 

## Tasks
- [ ] Remove content from tables and put into rich text field (Katherine)
- [ ] Edit changes (Laura)
- [ ] Copyedit (Megan)
- [ ] OCTO-DE review (Beth)

## Acceptance Criteria
- [ ] Content taken out of tables and put into rich text

**Comment by RLHecht** [comment]
@katherine-fung Here's the PDF for this page. This is another rates page that Beth reviewed already: [https://app.zenhub.com/files/133843125/f5663988-5b90-4020-ad64-3111bf27b45e/download](https://app.zenhub.com/files/133843125/f5663988-5b90-4020-ad64-3111bf27b45e/download)

**Comment by laurwill** [comment]
Thanks @katherine-fung ! Looks good — I made a tweak to those tricky "months" bullets. See what you think! If that makes sense to you, you can pass along to Megan for copyedit.

**Comment by katherine-fung** [comment]
Thanks @laurwill! 

@megzehn - This is ready for copyedit: http://preview-prod.vfs.va.gov/preview?nodeId=5449 

FYI I changed the effective date for the rates to "October 1, 2021" (instead of "October 1, 2022") to reflect the rates source page [https://www.benefits.va.gov/GIBILL/resources/benefits_resources/rates/ch1606/Ch1606rates100121.asp]. 

I also changed the academic year to say "2021 to 2022" (instead of 2022 to 2023). 

cc @bethpottsVADEPO The link above is the most recent rates page I've found, but please let us know if I missed something and we can change those years accordingly.

**Comment by bethpottsVADEPO** [comment]
@katherine-fung Are these the only rates where we couldn't find 2022-23?

**Comment by katherine-fung** [comment]
I haven't found 2022-23 for the MGIB-Active Duty rates either. They're not listed here [https://benefits.va.gov/gibill/resources/benefits_resources/rate_tables.asp], and searching within "site:va.gov" didn't turn a 2022-2023 page.

**Comment by RLHecht** [comment]
@katherine-fung I never put this as a task in the ticket, but could you do a double check of the rates on this page against the source page to make sure they're all correct. I think this was already done with the old content team but it would be good to do a double check. Thank you!

**Comment by katherine-fung** [comment]
@RLHecht Yes, I'll double check the rates against the source page!

**Comment by katherine-fung** [comment]
@RLHecht I confirmed the rates against the source page. I also added ".00" decimal amounts to the whole dollar rates, per Slack thread with Beth and Megan.

**Comment by katherine-fung** [comment]
Updating this ticket to reflect that I conferred with Laura ,and changed the style we use for the timeframes to say "1 to 6 months," "7 to 12 months," etc.

**Comment by RLHecht** [comment]
@megzehn Beth has this already so skipping your copyedit. You'll be doing a final copyedit on all these rates before we publish, so you'll still look through these. Thank you!

**Comment by RLHecht** [comment]
Closing this and will create a new ticket for final copyediting and publishing once the rates pages are ready.
